api/funkwhale_api/concerts/views.py

```diff
@@ -0,0 +1,42 @@
+import mux_python
+import os
+from dotenv import load_dotenv
+from rest_framework import viewsets
+from rest_framework.response import Response
+from rest_framework import status
+from .models import Concert
+from .serializers import ConcertSerializer
+
+# Load environment variables
+load_dotenv()
```

Comment:
This should already be happening through docker and django. I don't think we need this.